src/util-mpm-hs.c

```diff
@@ -570,6 +619,174 @@ static PatternDatabase *PatternDatabaseAlloc(uint32_t pattern_cnt)
     return pd;
 }
 
+static const char *HSCacheDefaultDir(void)
+{
+    static char hash_file_path[PATH_MAX];
+    int r = PathMerge(
+            hash_file_path, sizeof(hash_file_path), ConfigGetLogDirectory(), "hyperscan-cache");
```

Comment:
this should use `$localstatedir/cache/something`

Reply:
perhaps the dir should also be configurable

Reply:
as Jason mentioned, we need to make sure that dir doesn't conflict with suricata-update wrt permissions.

Reply:
and the code should be forgiving, right? :thinking:

Reply:
Yeah, just a warning on path errors I think